extractor/docker/docker.go

```diff
@@ -60,19 +60,29 @@ type layer struct {
 	Content io.ReadCloser
 }
 
-type opqDirs []string
-
 type DockerExtractor struct {
+	Client *client.Client
+	Cache  cache.Cache
 	Option types.DockerOption
 }
```

Comment:
Maybe say here `var _ Extractor  = DockerExtractor{}` to signal to user and compiler the intended use of the interface?

Reply:
Less is more.